build.gradle

```diff
@@ -78,6 +77,14 @@ subprojects {
         from javadoc
     }
 
+		task docs(type: Javadoc) {
```

Comment:
You're actually going to want this block to be outside of the `subprojects` block -- it should be in the "top level" of the tasks, so just put it at the very end of the file.